Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1418,6 +1418,9 @@
     <string name="errors_risk_detection_limit_reached_title">"Limit already reached"</string>
     <!-- XTXT: error dialog - Error description when the provideDiagnosisKeys quota limit was reached. -->
     <string name="errors_risk_detection_limit_reached_description">"No more exposure checks possible today, as you have reached the maximum number of checks per day defined by your operating system. Please check your risk status again tomorrow."</string>
+    <!-- XTXT: error dialog - Error description when the ssl certificate is deactivated on the device. -->
+    <string name="errors_ssl_certificate_deactivated">Bitte aktiviere das SYSTEM Sicherheitszertifikat T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2 auf ihrem Gerät. Mehr Informationen finden sie im FAQ auf https://coronawarn.app"</string>
```

Comment:
siehe oben

Reply:
This will be overwritten by the next translation delivery with the English version.